tests/README_INSTALL_TESTS.md

```
@@ -0,0 +1,24 @@
+# Running Installation Tests
+The tests in 	est_install_in_fresh_environment.py verify that the package dependencies are correctly specified.
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 14

(MD010, no-hard-tabs)